.github/workflows/release.yml

```diff
@@ -112,7 +112,7 @@ jobs:
         with:
           file: Dockerfile.release
           context: build
-          platforms: linux/amd64,linux/arm64v8
+          platforms: linux/amd64,linux/arm64/v8
           push: true
           tags: |
             fiberplane/am:v${{ needs.validate-version.outputs.version }}
```

Comment:
```suggestion
            fiberplane/am:v${{ needs.validate-version.outputs.version }}
            fiberplane/am:latest
```

Reply:
For now this is fine. There are some caveats when a hot fix will override a later version. It will also ensure that user will be fixed on a specific version without them realizing. But because of the ease of use and since it is a primarily local dev tool we can solve those issues later.